app/move/steps/create.js

```diff
@@ -180,6 +192,15 @@ module.exports = {
       'to_location_court_appearance',
     ],
   },
+  '/move-details-oca': {
```

Comment:
Same as with the field comment, I think we need to avoid using roles within logic and naming of things.

Is the rule that only people within OCA will be able to propose to move someone from one prison to another then?

Reply:
As far as I understand, yes -- only OCAs can `propose`. And we haven't got a solid way to identify this journey by location only. I am happy to rename it, but to what?